tests/unit/test_brailleDisplayDrivers.py

```diff
@@ -6,9 +6,45 @@
 """Unit tests for braille display drivers.
 """
 
+from brailleDisplayDrivers.seikantk import BrailleDisplayDriver as SeikaNotetakerDriver, SEIKA_INFO
+
 import unittest
 import braille
 
+
+class TestSeikaNotetakerDriver(unittest.TestCase):
+	def test_onReceive(self):
+		sampleCommand = SEIKA_INFO
+		sampleArgument = b"test"
+		sampleArgLen = bytes([len(sampleArgument)])
+		sampleMessage = sampleCommand + sampleArgLen + sampleArgument + b"\0\0\0"
+		paddedMessage = b"\0" + sampleMessage + b"\0"
+
+		class FakeSeikaNotetakerDriver(SeikaNotetakerDriver):
+			def __init__(self):
+				"""Sets the variables necessary to test _onReceive without a braille device
+				"""
+				self._hidBuffer = b""
+				self._command = None
+				self._argsLen = None
+
+			def _processCommand(self, command, arg):
+				"""Intercept processCommand to confirm _onReceive processes a message correctly.
+				"""
+				self._finalCommand = command
+				self._finalArg = arg
+				super()._processCommand(command, arg)
+		
+		stnk = FakeSeikaNotetakerDriver()
+		for i in range(len(sampleMessage)):
+			# the middle byte is the only one used, padded by 2 other bits
+			# paddedMessage[i:i + 2] == [sampleMessage[i-1], sampleMessage[i], sampleMessage[i+1]]

```

Comment:
can you remove this line?

Reply:
This is a continuation/clarification of the above comment.

```suggestion
			# i.e: paddedMessage[i:i + 2] == [sampleMessage[i-1], sampleMessage[i], sampleMessage[i+1]]
```

Reply:
nevermind my previous comment, removing